easybuild/easyconfigs/m/miRDeep2/miRDeep2-2.0.0.8-intel-2016b.eb

```diff
@@ -2,16 +2,16 @@ easyblock = 'PackedBinary'
 
 name = 'miRDeep2'
 version = '2.0.0.8'
-altver = '_'.join(version.split('.'))
+local_altver = '_'.join(version.split('.'))
 
 homepage = 'https://www.mdc-berlin.de/8551903/en/'
 description = "miRDeep2 is a completely overhauled tool which discovers microRNA genes by analyzing sequenced RNAs"
 
 toolchain = {'name': 'intel', 'version': '2016b'}
 
-source_url_subdir = 'research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/'
-source_urls = ['https://www.mdc-berlin.de/45995549/en/' + source_url_subdir]
-sources = ['mirdeep%s.zip' % altver]
+local_source_url_subdir = 'research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/'
+source_urls = ['https://www.mdc-berlin.de/45995549/en/' + local_source_url_subdir]
```

Comment:
could just inline this variable

Reply:
No, line too long :)

Reply:
Well, I would certainly always prefer a + and a newline when cutting up long strings in python than this procedure, but I also don't think this old obscure config is worth fighting over.
```
source_urls = ['https://www.mdc-berlin.de/45995549/en/' + 
               'research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/']
```